transport_plugins/bled112/iotile_transport_bled112/config_variables.py

```diff
@@ -9,5 +9,8 @@ def get_variables():
     conf_vars.append(["throttle-broadcast", "bool", "Only report changing broadcast values, not all values", "false"])
     conf_vars.append(["throttle-scan", "bool", "Only report device_seen once in a timeout", "false"])
     conf_vars.append(["throttle-timeout", "int", "device_seen and report events timeout in seconds", 30])
+    conf_vars.append(["deduplicate-broadcast-v2", "bool", "Filter unchanging Broadcast V2 packets per device", "false"])
```

Comment:
Since this will be a public api we can't change, I think we should use a cleaner naming scheme:

```
deduplicate-advertisements   # We're filtering all advertisements not just those containing broadcast data
deduplicate-timeout
```

Reply:
Well, we're not deduplicating all BLE advertisements, just the advertisements that include the bytes that identify them as an IOTile broadcast_v2 advertisement. I'm certainly open to a better name though.

Reply:
Maybe:
`throttle-v2-advertisements`
`throttle-v2-timeout`

One quick comment on `v1`, vs `v2`.  The version applies to the **advertisement packet format**, whether or not that advertisement contains a broadcast value or not.  So you can have a v2 advertisement without broadcasting anything and it would still be throttled by your code as I understand it.

Reply:
Ok, that sounds good.